src/assets/apiClient.js

```diff
@@ -53,4 +53,13 @@ export default class apiClient {
     const res = await this.authenticatedCall("get", `${url}recipe`);
     return res.data;
   }
+
+  addRecipe(
+    title, prep_time, cook_time, serves, ingredient, step) {
+    return this.autheticatedCall(
+      'post', url, { title,  prep_time, cook_time, serves, ingredient, step 
```

Comment:
You need to add to the url to get send the post request to the right end point. In this case the correct endpoint is https://mealplan-api.herokuapp.com/mealplan/add